src/modules/Blend/Module.js

```diff
@@ -34,7 +31,8 @@ module.exports = function Dynamic(options, UI, util) {
 
     getPixels(priorStep.output.src, function(err, pixels) {
       options.firstImagePixels = pixels;
-
+      // convert to runnable code:
```

Comment:
Let's always add standardized comments from now ie Capital letter at the start and so on...